content/webapp/services/prismic/fetch/index.ts

```diff
@@ -1,6 +1,5 @@
 import { Query, PrismicDocument } from '@prismicio/types';
 import * as prismic from '@prismicio/client';
-import fetch from 'node-fetch';
```

Comment:
Does this mean IE11 won't get client-side content. Do we mind if not. Or maybe does NextJS work some magic to make sure fetch is polyfilled?